fhir-ig-davinci-pdex-plan-net/src/main/java/com/ibm/fhir/ig/davinci/pdex/plan/net/util/IndexGenerator.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * (C) Copyright IBM Corp. 2020
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.ig.davinci.pdex.plan.net.util;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.FileReader;
+import java.io.OutputStream;
+
+import com.ibm.fhir.model.format.Format;
+import com.ibm.fhir.model.parser.FHIRParser;
+import com.ibm.fhir.model.parser.exception.FHIRParserException;
+import com.ibm.fhir.model.resource.Resource;
+import com.ibm.fhir.registry.util.Index;
+import com.ibm.fhir.registry.util.Index.Entry;
+
+public class IndexGenerator {
```

Comment:
Is it any different then our other IndexGenerators?  I'm wondering if this utility should be move to the base instead of copied to each IG.

Reply:
Or, if you keep a separate copy in each ig module, maybe move them to `src/test/java` so they don't go in the actual jar.

Reply:
I don't think it should go in tests. I will investigate making it a utility but there isn't a great place for it to go because it is reading from the local file system and anything outside of the project will lead to some wonky path resolution (though I'm sure it's doable).